canonical-web-and-design/charmhub.io
# Add functionality button

## Done
Added a functionality badge with a tooltip to charm cards

## QA
- Go to https://charmhub-io-975.demos.haus/
- Hover over the "Functionality" badge on the cards and check that there is a tooltip with a list with icons
